src/carousel/carousel.ts

```diff
@@ -147,18 +165,39 @@ export class NgbCarousel implements AfterContentChecked,
     this.showNavigationIndicators = config.showNavigationIndicators;
   }
 
+  @HostListener('mouseenter')
```

Comment:
Could you please move back these 2 `@HostListener` to the decorator's metadata ? When there is only one statement in the function, we usually put them directly up there.
→ `'(mouseenter)': '_mouseHover$.next(true)'`